source/family/nordic/nrf52/target.c

```diff
@@ -56,6 +56,23 @@ target_cfg_t target_device_nrf52_64 = {
     .target_part_number             = "nRF52832_xxAA",
 };
 
+// target information for nRF52840 with 256 KB RAM / 1024 KB Flash
+target_cfg_t target_device_nrf52840_256 = {
```

Comment:
It seems to me `target_device_nrf52840_256` is a duplicate of `target_device_nrf52840`.

Reply:
Resolved as per comment